tests/test_visitors/test_ast/test_naming/test_variable_self_reassignment.py

```diff
@@ -17,12 +19,34 @@
 test_variable = 10
 """
 
+wrong_fragment_double_assignment = """
+test_variable = 5
+test_variable = test_variable = 10
+"""
+
+wrong_fragment_other_assignment = """
+test_variable = 5
+test_variable = other = test_variable = 5
+"""
+
+wrong_fragment_tuple_assignment = """
+x = 1
+y = 2
+x, y = x, y
+"""
 
+
+@pytest.mark.parametrize('fragment', [wrong_fragment,
```

Comment:
We use different style for this kind of things: please, see https://github.com/wemake-services/wemake-python-styleguide/blob/56ce9b915404975f60f720363fe73e09b323cacc/tests/test_visitors/test_ast/test_naming/test_variable_usages.py#L37-L48